File: server/src/main/java/com/fluffy/exam/application/response/ExamDetailSummaryResponse.java
package com.fluffy.exam.application.response;

import com.fluffy.exam.domain.dto.AuthorDto;
import java.time.LocalDateTime;

public record ExamDetailSummaryResponse(
        Long id,
        String title,
        String description,
        String status,
        AuthorDto author,
        Long questionCount,
        Long likeCount,
        boolean isLiked,
        LocalDateTime createdAt,
        LocalDateTime updatedAt
) {
}

File: server/src/main/kotlin/com/fluffy/exam/application/response/ExamDetailSummaryResponse.kt
package com.fluffy.exam.application.response

import com.fluffy.exam.domain.dto.AuthorDto
import java.time.LocalDateTime

data class ExamDetailSummaryResponse(
    val id: Long,
    val title: String,
    val description: String,
    val status: String,
    val author: AuthorDto,
    val questionCount: Long,
    val likeCount: Long,
    val isLiked: Boolean,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime,
)
